docs/data/sdks/next-gen-sdks.md

```diff
@@ -0,0 +1,186 @@
+
+amplitude = Amplitude('API_KEY')
+```
+
+### Configuration
+
+Configuration includes the options to configure Client behavior. 
+
+You can pass a Configuration on Client initialization.
+
+```python
+# Next-gen Python SDK example: initialize a Client with a Configuration
+
+from amplitude import Amplitude
+from amplitude import Config
+
+amplitude = Amplitude("API_KEY", 
+                      configuration=Config(
+                            flush_queue_size=100, # flush queued events when there are 100 or more
+                            flush_interval_millis=20000 # flush queued events every 20 seconds
+                    )) 
+```
+
+You can also get access to the Configuration of your Client after initialization.
+
+```python
+# Next-gen Python SDK example: get access to the Configuration of your Client
+
+from amplitude import Amplitude
+
+client = Amplitude('API_KEY')
+# flush queued events when there are 100 or more
+client.configuration.flush_queue_size = 100
+# flush queued events every 20 seconds
+client.configuration.flush_interval_millis = 20000
+```
+
+### Storage provider
+
+A storage provider holds events in a designated storage buffer. The default storage provider only queues events in memory. To prevent data loss, you can customize your own storage provider to persist events, for example to a disk, in case of unexpected shutdown. This way, you can ensure that you can recover un-sent events even in the event of a system failure. 
+
+You can configure the storage provider in the Client Configuration. Refer to the "Configuration Options" section of a specific next-gen SDK documentation for more details.
+
+### Logger provider
+
+A logger provider configures the logger instance used by the Amplitude Client. It helps to collect debugging and error messages from the SDK in both development and production environments.
+
+You can configure the logger provider in the Client Configuration. Refer to the "Configuration Options" section of a specific next-gen SDK documentation for more details.
+
+### Event 
+
+Events represent how users interact with your application. 
+
+`BaseEvent` represents a basic event with optional properties. You can also track other event properties in the field of `EventProperties`
+
+```python
+# Next-gen Python SDK example: Event
+
+from amplitude import Amplitude, BaseEvent
+
+client = Amplitude('API_KEY')
+
+# Track a basic event
+# One of user_id and device_id is required
+event = BaseEvent(event_type="Button Clicked", user_id="USER_ID")
+client.track(event)
+
+# Track events with optional properties
+client.track(
+    BaseEvent(
+        event_type="Button Clicked",
+        user_id="USER_ID",
+        app_version="1.2.1" # optional, the current version of your application
+        event_properties={ # other event properties you want to track 
+            "color": "red"
+        }
+))
+```
+
+### Plugin
+
+Plugins allow you to extend Amplitude SDK's behavior by, for example:
```

Comment:
I think we should say maintenance SDKs using middleware, and next-gen use Plugins is the replacement. I think it is the good place to show the structure of the payload to let the customer know which they can update while using the plugin.

Reply:
I think it might be better to have a dedicated page about middleware that we can link to from here.

Reply:
Put this to "Comparison with maintenance SDK" section and leave this section clean to introduce only plugins